src/pages/index.js

```diff
@@ -36,8 +36,9 @@ export { NotFound } from './NotFound/NotFound'
 export { SetPassword } from './SetPassword/SetPassword'
 export { WalletsStart } from './WalletsStart/WalletsStart'
 export { AgreementsView } from './Agreements/AgreementsView'
-
+export { IntroductionView } from './Introduction/IntroductionView'
```

Comment:
should be named just `Introduction`

Reply:
Why?

Reply:
Naming convention, see here: https://jibrelnetwork.atlassian.net/wiki/spaces/~40436670/pages/803766412/Routes

Reply:
Rename all files and lines as "Introduction"?

Reply:
We agreed on naming as follows:
- if the page requires separation to connector and view, we name view `SomethingView` and connector (cause it is actual page) – `Something`
- if the page does not require separation, we name the only file as `Something`

Everything correct, @IViol ?